packages/vuetify/src/directives/touch/__tests__/touch.spec.ts

```diff
@@ -0,0 +1,116 @@
+// Directives
+import Touch from '../'
+
+// Libraries
+import Vue from 'vue'
+
+// Utilities
+import {
+  mount,
+  Wrapper
+} from '@vue/test-utils'
+
+// Simulate touch
+function touch (element) {
```

Comment:
Haven't we got touch function in test folder?

Reply:
Yes it is in the "Points of Attention" section in my PR :)

My understanding was that the intent is to have everything self contained within their respective test folders (directives, components…) and that this file would eventually be removed.

Reply:
No, test folder won't be removed. I'm working on avoriaz removal now.